### ISSUE TITLE
Still Not Able To Make Image Alt Appear As Caption

### ISSUE THREAD

**Issue by d80wrk** [liferay/alloy-editor#1532]
**Do you want to request a _feature_ or report a _bug_?**
Feature Request

**What is the current behavior?**
Cannot make alt text appear as a caption (transform the image into a figure)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Al text can be added but missing checkbox to make alt a caption

**What is the expected behavior?**
Feature to add a caption to an image implemented.

**Which versions of alloy-editor, and which browser / OS are affected by this issue? Did this work in previous versions?**
All, no it has never worked.

![missing-checkbox-to-make-alt-a-caption](https://user-images.githubusercontent.com/73597832/175098763-f788c9cc-555d-4d8a-97e1-82e954627670.png)
